### PR TITLE
Feature/ability to delete a custom report 123130571

### PR DESCRIPTION
**Pull request by pashgo**
https://www.pivotaltracker.com/story/show/123130571

### CODE REVIEW

**Comment by cheezy2022** [comment]
🆒

File: lib/gap_intelligence/client/downloads.rb [hunk 1]
```diff
@@ -28,5 +28,18 @@ def downloads(owner_id, params = {}, options = {}, &block)
...
       perform_request(:get, 'downloads', options.merge(params: params.merge(owner_id: owner_id)), &block)
     end
+
+    # Deletes download
+    #
+    # @param owner_id [String,Integer] owner id of the download
+    # @param download_ids [Integer] id of the download
+    # @param options [Hash] options the options to make the request with
+    # @yield [request] The Faraday request
+    # @return [Download] the created download
```
Comment (shavkat-samatov) [inline lib/gap_intelligence/client/downloads.rb]: I think this is a tiny typo in `created download`

Comment (shavkat-samatov) [inline lib/gap_intelligence/client/downloads.rb]: I just checked gAPI side and actually saw a returning 2xx status `head :no_content`, not actual Download instance

**Comment by shavkat-samatov** [comment]
other than small comment, looks good to me! 👍

File: lib/gap_intelligence/client/downloads.rb (hunk 1 above)
Comment (shavkat-samatov) [inline lib/gap_intelligence/client/downloads.rb]: update: just noticed in https://github.com/GapIntelligence/gapi/pull/1068/files that endpoint actually returns `render json: { data: { status: 'OK' } }`, not actual Download instance

**Pull request merged**